packages/components/src/Lettera/Models.purs

```diff
@@ -413,6 +413,7 @@ type BoxInfo =
   { title :: Maybe String
   , headline :: Maybe String
   , content :: Array String
+  , type :: Maybe String
```

Comment:
We should always `type` present. According to lettera `title` too actually.

Reply:
(Although didn't check your changes in lettera regarding this yet)

Reply:
Yes, I was going to remove the Maybe when Lettera gets updated. The current version of Lettera doesn't have the type field, it would fail all articles with any box blocks if I did it already.

Reply:
Lettera has been updated to have the type field, I updated this to remove the Maybe.